target/linux/realtek/image/rtl838x.mk

```diff
@@ -27,6 +27,14 @@ define Device/d-link_dgs-1210
 	dlink-version | dlink-headers
 endef
 
+define Device/d-link_dgs-1210-10mp
+  $(Device/d-link_dgs-1210)
+  SOC := rtl8380
+  DEVICE_MODEL := DGS-1210-10MP
```

Comment:
Please add `DEVICE_VARIANT := F`.

Only the recent family revisions use Realtek SoCs, so I think we should be specific about this one.